src/sql/workbench/services/notebook/browser/models/notebookModel.ts

```diff
@@ -135,27 +135,43 @@ export class NotebookModel extends Disposable implements INotebookModel {
 		this._defaultKernel = _notebookOptions.defaultKernel;
 	}
 
-	public get notebookManagers(): INotebookManager[] {
-		let notebookManagers = this._notebookOptions.notebookManagers.filter(manager => manager.providerId !== DEFAULT_NOTEBOOK_PROVIDER);
-		if (!notebookManagers.length) {
-			return this._notebookOptions.notebookManagers;
+	private get serializationManagers(): ISerializationManager[] {
+		let managers = this._notebookOptions.serializationManagers.filter(manager => manager.providerId !== DEFAULT_NOTEBOOK_PROVIDER);
+		if (!managers.length) {
+			return this._notebookOptions.serializationManagers;
 		}
-		return notebookManagers;
+		return managers;
 	}
 
-	public get notebookManager(): INotebookManager | undefined {
-		let manager = this.notebookManagers.find(manager => manager.providerId === this._providerId);
+	public get serializationManager(): ISerializationManager | undefined {
+		let manager = this.serializationManagers.find(manager => manager.providerId === this._providerId);
+		if (!manager) {
+			manager = this.serializationManagers.find(manager => manager.providerId === DEFAULT_NOTEBOOK_PROVIDER);
+		}
+		return manager;
+	}
+
+	public get executeManagers(): IExecuteManager[] {
```

Comment:
Should this be private like `get serializationManagers()`?

Reply:
No, this gets used in a couple of other places.